### DISCUSSION TITLE
Is 'prefix' supported?

### DISCUSSION THREAD

**Discussion by brankoconjic** [dcastil/tailwind-merge#77] [Help] [answered]
Hi,

Is [prefix](https://tailwindcss.com/docs/configuration#prefix) option supported?

Thanks!

**Comment by dcastil** [comment]
Hey @brankoconjic! 👋 

At the moment it isn't supported unfortunately. But I think I could add that feature if you need it. Do you use the prefix feature in your Tailwind config?

**Reply by brankoconjic** [reply]
Yes, I do. That'd be amazing, thanks again.

**Reply by dcastil** [reply]
Okay, I'll do it in the coming days and notify you here when it's ready. 😊

**Comment by dcastil** [comment, marked as answer]
I just released [v1.2.0](https://github.com/dcastil/tailwind-merge/releases/tag/v1.2.0) with prefix support. You can set it in both `extendTailwindMerge` and `createTailwindMerge`.

If you use the default Tailwind config with a prefix, you can use it like this:

```ts
import { extendTailwindMerge } from 'tailwind-merge'

export const twMerge = extendTailwindMerge({ prefix: 'my-prefix-' })
```
